internal/version/version.go

```diff
@@ -0,0 +1,56 @@
+package version
+
+import (
+	"errors"
+	"runtime/debug"
+)
+
+var (
+	Version string
+)
+
+type VCSInfo struct {
+	VCS          string
+	Revision     string
+	Date         string
+	TreeModified string
+}
+
+type Info struct {
+	Version   string
+	VCSInfo   VCSInfo
+	GoVersion string
+	Compiler  string
+	Platform  string
+}
+
+func GetBuildInfo() (*Info, error) {
```

Comment:
Please rename this to `BuildInfo` so it is inline with the suggestions [here](https://go.dev/doc/effective_go#Getters).

```suggestion
func BuildInfo() (*Info, error) {
```